### ISSUE TITLE
render method error missing return type annotation

### ISSUE THREAD

**Issue by craigcosmo** [gajus/eslint-plugin-flowtype#189]
Is there way to suppress this?

<img width="1438" alt="screen shot 2017-01-18 at 11 47 21 am" src="https://cloud.githubusercontent.com/assets/538657/22051228/03744c98-dd74-11e6-8f67-5c4e21c29881.png">

**Comment by danharper** [comment]
You should be able to use the [`"excludeMatching": ["render"]`](https://github.com/gajus/eslint-plugin-flowtype/blob/master/.README/rules/require-return-type.md) option. (For some reason it's missing from the main readme).

**Comment by stevenjmarsh** [comment]
I spotted this as well. And did a little testing. It 'seems' that "excludeMatching" does properly identify top level functions and arrow functions and exclude them (not indicate warning). However does not identify class methods.

I created a simple top level function without a return type annotation, excluded it, and eslint ignored it.

What's interesting is, it can recognize the class method to identify a return type annotation does not exist, but does not successfully do so when trying to exclude it.
